actor/src/test/scala/com/evolutiongaming/akkaeffect/ActorVarTest.scala

```diff
@@ -53,6 +53,9 @@ class ActorVarTest extends AsyncFunSuite with Matchers {
 
     def actorVar(stop: F[Unit]) = Sync[F].delay {
       val act = new Act[F] {
+
+        def postStop(): Unit = {}
```

Comment:
here and in other places - IMHO we should use explicit Unit like `()` instead of implicit return value of empty block `{}`

Reply:
Please elaborate, I see no reason in using `()`

Reply:
`()` is explicit value, while `{}` is block of expressions (though empty), which has implicit default return value, which happens to be `Unit`.

from compilation perspective, resulting code in Java bytecode will be the same, but from reading perspective, explicit value makes more sense to me.

from [docs](https://www.scala-lang.org/api/current/scala/Unit.html):

> Unit is a subtype of [scala.AnyVal](https://www.scala-lang.org/api/current/scala/AnyVal.html). There is only one value of type Unit, `()`, and it is not represented by any object in the underlying runtime system.

Reply:
So why should I prefer `()` instead of `{}`?

Reply:
for me `{}` looks like:
```scala
def get4: Int = 1 + 3 // here developer and compiler has calculate
```
instead of 
```scala
def get4: Int = 4
```

Reply:
Ok, got u. Could we also use 
```
align.preset=most
```
because for me it looks nicer?

Reply:
the `style = defaultWithAlign` was set by you on 2023-12-13 and `align.preset = most` would bring in so many non-business changes in this PR. 

reformatting deserves its own PR, though for **me** following style looks ugly 😄 
```sbt
      Akka.slf4j                  % Test,
      Akka.testkit                % Test,
      Cats.core,
      CatsEffect.effect,
      `cats-helper`,
      pureconfig,
      smetrics,
      scalatest                   % Test,
      `akka-persistence-inmemory` % Test,
```